### ISSUE TITLE
notebook blank

### ISSUE THREAD

**Issue by zapcity** [spyder-ide/spyder-notebook#324]
The notebooks are blank. I'm suspecting the jupyter notebook service is not starting? Is jupyter suppose to be started manually before starting spyder? Or is it started when spyder starts? I also tried with spyder 4.1.0 and same deal. With spyder 3.3.6 at least it's giving me an error saying that jupyter notebook cannot start, so i suspect that launching spyder should automatically launch jupyter.

**What steps will reproduce the problem?**

1. conda create -n dev python=3.7
2. activate dev
3. conda install spyder=4.1.5
4. conda install spyder-notebook -c spyder-ide
5. start spyder
6. notebooks are blank

**What is the expected output? What do you see instead?**

I expect jupyter notebook to start when i start spyder and the notebook cells to be visible. But they are all blank

![image](https://user-images.githubusercontent.com/37337474/96332570-1a7ecd80-10a8-11eb-82db-5b3555a0f1c6.png)

**Please provide any additional information below (stack trace, screenshots, a gif, etc)**

used conda to install with python 3.7

## Versions and main components

* Spyder Version: 4.1.5
* Python Version: 3.7.8
* Qt Version: 5.12.9
* PyQt Version: 5.12.3
jupyter core     : 4.6.3
jupyter-notebook : 6.1.4
qtconsole        : 4.7.7
ipython          : 7.18.1
ipykernel        : 5.3.4
jupyter client   : 6.1.7
jupyter lab      : not installed
nbconvert        : 6.0.7
ipywidgets       : not installed
nbformat         : 5.0.8
traitlets        : 5.0.5
* Operating system: windows 10 2004

## Dependencies

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.6.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.18.1 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  6.0.7 (OK)
numpydoc >=0.6.0               :  1.1.0 (OK)
paramiko >=2.4.0               :  2.7.2 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.2 (OK)
pygments >=2.0                 :  2.7.1 (OK)
pylint >=1.0                   :  2.6.0 (OK)
pyls >=0.34.0;<1.0.0           :  0.35.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  1.0.1 (OK)
qtconsole >=4.6.0              :  4.7.7 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.2.1 (OK)
spyder_kernels >=1.9.4;<1.10.0 :  1.9.4 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.2 (OK)

# Optional:
cython >=0.21                  :  None (OK)
matplotlib >=2.0.0             :  None (OK)
numpy >=1.7                    :  None (OK)
pandas >=0.13.1                :  None (OK)
scipy >=0.17.0                 :  None (OK)
sympy >=0.7.3                  :  None (OK)

# Spyder plugins:
spyder_notebook                :  0.3.0 (OK)

**Comment by gromanmartin** [comment]
I had the same problem, but realized I was missing jupyter package in my conda env.

**Comment by Enrico68** [comment]
Same problem here !. @gromanmartin :

```
conda install jupyter
Collecting package metadata (current_repodata.json): done
Solving environment: done

# All requested packages already installed.
```

**Comment by jitseniesen** [comment]
You should not need to start anything manually. This is probably the same as discussed at the bottom of #314, so try to downgrade traitlets to version 4 with `conda install traitlets=4`.

If that does not help, have a look at the server log. You can view the server log from the option (hamburger) menu in the Notebook pane. If the log does not help you find the problem, please send it to us and we will see what we can do.

**Comment by AJSchiller** [comment]
@jitseniesen that solved it for me, thanks!

**Comment by Enrico68** [comment]
solved for me top, thanks to @jitseniesen

**Comment by jitseniesen** [comment]
Excellent, thanks for the confirmation. I am closing this as a duplicate of #314.
